package br.ufpb.iago.backend;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;

@SpringBootApplication
@EnableCaching
public class BackendAttractionsApplications {
    public static void main(String[] args) {
        SpringApplication.run(BackendAttractionsApplications.class, args);
    }
}